packages/react-components/stories/systems-summary.stories.tsx

```diff
@@ -0,0 +1,159 @@
+  SystemSummaryAlertProps,
+  SystemSummaryAlert,
+  SystemSummaryTaskState,
+  SystemSummaryTaskStateProps,
+  SystemSummaryBanner,
+  SystemSummaryBannerProps,
+  SystemSummarySpoiltItems,
+  SystemSummarySpoiltItemsProps,
+  Severity,
+} from '../lib';
+import { Meta, Story } from '@storybook/react';
+
+export default {
+  title: 'Systems summary',
+  component: SystemSummaryItemState,
+} as Meta;
+
+const itemStateDataDoor: SystemSummaryItemStateProps = {
+  itemSummary: {
+    item: 'Doors',
+    itemSummary: { operational: 1, outOfOrder: 1 },
+  },
+  onClick: () => {
+    /**filler */
+  },
+};
+
+const itemStateDataRobot: SystemSummaryItemStateProps = {
+  itemSummary: {
+    item: 'Robots',
+    robotSummary: { operational: 2, outOfOrder: 0, idle: 1, charging: 1 },
+  },
+  onClick: () => {
+    /**filler */
+  },
+};
+
+const systemSummaryAlert: SystemSummaryAlertProps = {
+  notifications: [
+    { id: 1, severity: Severity.High, time: 'January 29th 2021, 8:20:50', error: 'hello world' },
+  ],
+};
+
+const systemSummaryTaskStateData: SystemSummaryTaskStateProps = {
+  tasks: [
+    {
+      task_id: 'abc',
+      state: 0,
+      start_time: { sec: 0, nanosec: 0 },
+      end_time: { sec: 0, nanosec: 0 },
+      status: 'status',
+      submission_time: { sec: 0, nanosec: 0 },
+    },
+    {
+      task_id: 'abc',
+      state: 1,
+      start_time: { sec: 0, nanosec: 0 },
+      end_time: { sec: 0, nanosec: 0 },
+      status: 'status',
+      submission_time: { sec: 0, nanosec: 0 },
+    },
+    {
+      task_id: 'abc',
+      state: 2,
+      start_time: { sec: 0, nanosec: 0 },
+      end_time: { sec: 0, nanosec: 0 },
+      status: 'status',
+      submission_time: { sec: 0, nanosec: 0 },
+    },
+    {
+      task_id: 'abc',
+      state: 3,
+      start_time: { sec: 0, nanosec: 0 },
+      end_time: { sec: 0, nanosec: 0 },
+      status: 'status',
+      submission_time: { sec: 0, nanosec: 0 },
+    },
+  ],
+  onClick: () => {
+    /**filler */
+  },
+};
+
+const systemSummaryBannerData: SystemSummaryBannerProps = {
+  isError: false,
+};
+
+const spoiltEquipment: SystemSummarySpoiltItemsProps = {
+  doors: [
+    {
+      door: {
+        name: 'hardware_door',
+        v1_x: 4.9,
+        v1_y: -4,
+        v2_x: 4.4,
+        v2_y: -5,
+        door_type: 1,
+        motion_range: 1.571,
+        motion_direction: -1,
+      },
+      itemNameAndState: 'door - unknown',
+    },
+  ],
+  lifts: [],
+  dispensers: [],
+  robots: [],
+};
+
+export const SystemSummaryItemStateStory: Story = (args) => (
+  <React.Fragment>
+    <SystemSummaryItemState
+      itemSummary={itemStateDataDoor.itemSummary}
+      onClick={itemStateDataDoor.onClick}
+      {...args}
+    />
+    <SystemSummaryItemState
+      itemSummary={itemStateDataRobot.itemSummary}
+      onClick={itemStateDataRobot.onClick}
+      {...args}
+    />
+  </React.Fragment>
+);
+
+export const SystemSummaryItemAlertStory: Story = (args) => (
```

Comment:
This will be an api call to the backend once it is up. I have removed it for now since the backend is not up yet. Will a TODO comment for now

Reply:
note that this being in `react-components`, it must not depend on any backend. There is currently no way for the story to handle the button because there is no handler props it can use, the story should be able to perform the needed actions even without a backend.

It can do so by using something along the lines of
```tsx
<SystemSummaryAlert notifications={notifications} onNotificationDismiss={(id) => setNotifications((prev) => prev.filter((x) => x.id !== id)} {...args} />
```

On the actual dashboard, `onNotificationDismiss` will do whatever is needed to keep things in sync (might include updating the db etc).